### PR TITLE
Feature/aps 2080 update venue models

### PR DESCRIPTION
**Pull request by TanyaEvtushenko**

### CODE REVIEW

File: EncoreTickets.SDK/Api/Utilities/EnvironmentHelper.cs [hunk 1]
```diff
@@ -13,20 +14,22 @@ public static class EnvironmentHelper
...
-                case "prod":
-                    return Environments.Production;
-                default:
-                    throw new ArgumentException($"Environment {name} is not defined.");
-            } 
+                return EnumExtension.GetEnumFromString<Environments>(name);
+            }
+            catch (Exception)
+            {
+                switch (name.ToLower())
```
Comment (YuryStruchkou) [inline EncoreTickets.SDK/Api/Utilities/EnvironmentHelper.cs]: Why is this change needed? Seems much less readable and more error-prone than it was before

Comment (TanyaEvtushenko) [reply EncoreTickets.SDK/Api/Utilities/EnvironmentHelper.cs]: This was done to expand the available string values ​​with values ​​that correspond to the enum values ​​by their names. For example, in the previous version it was impossible to get enum for the string “sandbox”

Comment (YuryStruchkou) [reply EncoreTickets.SDK/Api/Utilities/EnvironmentHelper.cs]: Is "sandbox" string even used anywhere outside the SDK? Previously we had one switch-case statement in this method, now we have try-catch blocks with the catch block containing part of the old switch-case. It has become less readable

Comment (TanyaEvtushenko) [reply EncoreTickets.SDK/Api/Utilities/EnvironmentHelper.cs]: OK, I can get it back. I agree that this is less readable, but it is more flexible and will continue to work if new environments are added

Comment (YuryStruchkou) [reply EncoreTickets.SDK/Api/Utilities/EnvironmentHelper.cs]: It would break if an enum entry gets renamed though unlike the previous solution so I would argue that it's even not really more flexible
